Sources/TuistCore/Graph/Graph.swift

```diff
@@ -464,6 +472,10 @@ extension Graph {
             return isDynamicLink ?? false
         }
     }
+	
+    func isXCFramework(node: GraphNode) -> Bool {
```

Comment:
I'd not create a method for this because the wrapped logic is very short and simple.

Reply:
i could define it in the closure itself